acceptance-tests/lib/testState.ts

```diff
@@ -0,0 +1,65 @@
+import { getTestConfig } from './env';
+import { createCli } from './cmd';
+import { CLI, TestConfig } from './types';
+import { getInitPromptSequence } from './prompt';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import yaml from 'js-yaml';
+import rimraf from 'rimraf';
+import { v4 as uuidv4 } from 'uuid';
+
+export class TestState {
```

Comment:
This is a helper class to manage all of the state needed to be able to run the tests.  It helps to segment the tests so they all have their own copy of the test state and can run in isolation from one another.  It handles the generation of a config file with a semi-randomized name so file system collisions should not happen between tests.